flake.nix

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Is it possible to avoid placing flake.nix or flake.lock in the root of the repository? I want to reduce the number of files in the repo root if possible. (default.nix is already annoying IMO...)

Reply:
I will say it will defeat the purpose of flakes.
the way I would like to look it as flakes are used at the "edge" of the repo to "report" what it contains.
it should be top level by design. for people that run:
```
nix run quick-lint/quick-lint-js 
#or
nix show quick-lint/quick-lint-js 
```
to easily find the implementations.
so you have guaranty that it will be the top level flake.nix